packages/firestore/src/util/log.ts

```diff
@@ -19,37 +19,42 @@
 import { SDK_VERSION } from '../core/version';
 import { AnyJs } from './misc';
 import { PlatformSupport } from '../platform/platform';
+import { Logger, LogLevel as _LogLevel } from '@firebase/logger';
+
+const client = new Logger();
```

Comment:
client => firebaseLogger? (or at least logClient or similar)

Reply:
`logClient` is the name used in `packages/database/src/core/util/util.ts`.

Reply:
SGTM